### PR TITLE
feat(Cache): add `entries` and `values` combinators.

### PR DESCRIPTION
**Pull request by patroza**

### CODE REVIEW

File: packages/cache/src/Cache/index.ts [hunk 1]
```diff
@@ -465,6 +480,14 @@ export function size<Key, Error, Value>(self: Cache<Key, Error, Value>): T.UIO<n
   return self.size
 }
 
+/**
+ * Returns the approximate values in the cache.
+ */
+ export function values<Key, Error, Value>(self: Cache<Key, Error, Value>): T.UIO<readonly Value[]> {
```
Comment (IMax153) [inline packages/cache/src/Cache/index.ts:486]: Can we add a corresponding test for the new `values` combinator?

**Review by IMax153 — COMMENTED** [review]
Thank you for the PR @patroza ! 

Looks pretty good to me - just one comment and one request 🙏 

Would you mind also removing the `CAache` abstract class entirely? It is a remnant of a previous refactor that should not have made it into the release 😄

**Comment by IMax153** [comment]
Also - it looks like prettier may be failing in CI-CD:

```
[@effect-ts/cache::yarn]   274:37  error  Replace `·A.filterMap_([...cacheState.map.values()],·(v)·=>·v._tag·===·"Complete"·&&·v.exit._tag·===·"Success"·?·O.some(v.exit.value)·:·O.none)` with `⏎············A.filterMap_([...cacheState.map.values()],·(v)·=>⏎··············v._tag·===·"Complete"·&&·v.exit._tag·===·"Success"⏎················?·O.some(v.exit.value)⏎················:·O.none⏎············)⏎··········`  prettier/prettier
[@effect-ts/cache::yarn]   486:1   error  Replace `·export·function·values<Key,·Error,·Value>(self:·Cache<Key,·Error,·Value>` with `export·function·values<Key,·Error,·Value>(⏎··self:·Cache<Key,·Error,·Value>⏎`                                                                                                                                                                                                           prettier/prettier
[@effect-ts/cache::yarn]   489:1   error  Delete `·`                                                                                                                                                                                                                                                                                                                                                                        prettier/prettier
```

You should be able to run `yarn autofix` to take care of this 👍

**Comment by patroza** [comment]
@IMax153 thanks! yea interestingly my vscode I not auto fixing in this repo :o

**Review by IMax153 — APPROVED** [review]
LGTM 👍

**Comment by IMax153** [comment]
We should probably still add a test for this combinator and the one in #11.

File: packages/cache/src/Cache/index.ts (hunk 1 above)
Comment (patroza) [reply packages/cache/src/Cache/index.ts]: You got it!

**Comment by patroza** [comment]
@IMax153 ready, also added `entries` combinator.

**Review by IMax153 — APPROVED** [review]
LGTM 👍 - thanks for your work on this

**Pull request merged**